CrowdSaleDZT.scilla

```diff
@@ -99,6 +112,40 @@ procedure IsOwner()
   end
 end
 
+procedure sendReferral(refAddr: ByStr20, amount: Uint128)
+  is_ref = builtin eq zero_address refAddr;
+
+  match is_ref with
+  | True =>
+  | False =>
+    referral_percent <- referralPercent;
+    max_percent <- max_referral_percent;
+    some_refferal_percent <- referrals[refAddr];
+
+    percent = get_percent  some_refferal_percent referral_percent;
+
+    is_max = builtin eq percent max_percent;
+
+    match is_max with
+      | False =>
+        new_referrals_percent = builtin add percent one128;
+        referrals[refAddr] := new_referrals_percent
+      | True =>
+    end;
+
+    reward = builtin div amount one_hundred;
+    reward = builtin mul reward percent;
```

Comment:
We need to change operation order. Multiplication first and division second. (lines 136-137)